### PR TITLE
Remove automatic updates from lock package

### PR DESCRIPTION
**Pull request by ChaituVR**
Right now updating a GitHub package with yarn (lock repo) updates all the packages on yarn.lock, so for now it is better to gt updates only from snapshot.js

#1506

### CODE REVIEW

**Comment by samuveth** [comment]
Not sure I understand what this fixes. How would be update lock.js in the future with this change?

**Comment by mktcode** [comment]
Shouldn't we just publish the package normally and instead of installing from the GitHub repo, installing from npm registry? Like... well... everyone else? 🤪 I mean, the argument for installing directly from master, is our high frequency of updates but I'd rather have a version number like 1.3.434756357 than dealing with the inconvenience of installing un-built raw github repos.

File: .github/workflows/update-snapshot-packages.yml [hunk 1]
```diff
@@ -15,10 +15,7 @@ jobs:
           node-version: '14.x'
       - name: Update snapshot.js
         run: |
-          yarn upgrade @snapshot-labs/snapshot.js  --latest
-      - name: Update snapshot lock
-        run: |
-          yarn upgrade @snapshot-labs/lock@github:snapshot-labs/lock#master  --latest
```
Comment (samuveth) [inline .github/workflows/update-snapshot-packages.yml:21]: @ChaituVR I think we should publish a new version for lock.js and remove only `@github:snapshot-labs/lock#master`.

Comment (ChaituVR) [reply .github/workflows/update-snapshot-packages.yml:21]: @samuveth Can we merge this for now? It is blocking for snapshot.js to add new networks, will create a new PR with lock.js from the npm package

**Review by samuveth — APPROVED** [review]

**Comment by samuveth** [comment]
@ChaituVR Done

**Pull request merged**